apps/fabric-website/src/SiteDefinition/SiteDefinition.tsx

```diff
@@ -58,18 +59,15 @@ export const SiteDefinition: ISiteDefinition<Platforms> = {
   ],
   messageBars: [
     {
-      path: '#/controls/web',
-      text: 'Fabric 7 is now released with the new Fluent styles.',
+      path: '#',
+      text: (
+        <span>
+          <strong style={{ fontWeight: 700 }}>Coming soon</strong> UI Fabric is evolivng into Fluent UI.
```

Comment:
Doesn’t strong do this automatically?

Reply:
it only did weight 600, which wasn't very different